deeposlandia/webapp.py

```diff
@@ -230,22 +231,24 @@ def demo_image_selector():
     summarizes the label information for displyaing purpose
     """
     dataset = request.args.get('dataset')
-    server_folder = os.path.join("deeposlandia", "static", dataset, "images")
-    client_folder = os.path.join("/", "static", dataset, "images")
+    dataset_code = dataset + "_agg" if dataset == "mapillary" else dataset
+    server_folder = os.path.join("deeposlandia", "static", dataset_code, "images")
+    client_folder = os.path.join("/", "static", dataset_code, "images")
```

Comment:
I think the STATIC Flask folder is defined in a Flask configuration variable. See http://flask.pocoo.org/docs/1.0/api/?highlight=static%20folder#flask.Flask.static_folder

Reply:
Two modifications:
- os.path.join("deeposlandia", "static") -> app.static_folder
- os.path.join("/", "static") -> app.static_url_path